### DISCUSSION TITLE
Bug report: StackOverflow positioning

### DISCUSSION THREAD

**Discussion by ctjlewis** [lowlighter/metrics#683] [General]
Hey Simon, no idea where to start debugging this but if you can point me in the right direction I might be able to help.

![image](https://user-images.githubusercontent.com/1657236/143729864-ebc2199a-5fa9-4d7d-bec4-174d34937b90.png)

**Comment by lowlighter** [comment]
Hum I wonder if it's related to the fact that xml is pretty printed.
Since these are in `<code>` tags which preserves white-space, it probably keeps the leading white space

**Comment by lowlighter** [comment]
Sorry for the late response but this has been patched a few days ago by #755 👍
